lib/HLSL/DxilPreparePasses.cpp

```diff
@@ -197,7 +197,7 @@ static Function *StripFunctionParameter(Function *F, DxilModule &DM,
   FunctionType *FT = FunctionType::get(VoidTy, false);
   for (auto &arg : F->args()) {
     if (!arg.user_empty())
-      return nullptr;
+      return nullptr; // If this would return here, we really should produce an error somewhere.
```

Comment:
This comment was intended to identify something we needed to do before merging the changes I had (since it silently fails to remove arguments if they are still used).  I didn't intend to just add the comment here.